### PR TITLE
NR-66464: Worked on PHP v2 enhancement

### PR DESCRIPTION
**Pull request by dikshab0202**
Ticket: https://issues.newrelic.com/browse/NR-66464

Description: PHP v2 enhancements, worked on below points:

Updated 'About' Widget with this detail -> **Unable to find data in your dashboard?** 

Updated Work:
-> Removed the mentioned content from about widget : " Detect common installation and configuration problems "
-> updated logo url
-> removed filter before capturing json

1. Worked on separating logo and below content to the different widgets and also resized the logo.
2. Worked on separating different types of monitoring into different tabs so that vertical scroll can be avoided.
3. Removed duplicate unnecessary widgets.
4. Worked on complicated widgets to make it simple.
5. Removed filter part from specific widgets.
6. Renamed the first tab - 'PHP' to 'overview'
7. Added tab for alerts
8. Placed graphs and charts in the middle.
9. Removed 'app name' and other conditions from queries
10. Modified the description of browser monitoring
11. Added 'About' heading to information widget
12. Capturing screenshot and json to work on PR

Updated first image after updating 'about' widget:
php01

Updated first image after removing content - "Detect common installation and configuration problems with New Relic agents. Optionally, attach this troubleshooting data to a support case. Learn more."

php01
---------

php01

php02

php03

php04

php05

# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include an InstallPlan and Documentation reference?
- [ ] Are all documentation links publically accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)? 
- [ ] Did you check your descriptive content for spelling and grammar errors?

### Dashboards 

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: dashboards/php/php.json [hunk 1]
```diff
@@ -12,23 +12,23 @@
...
-			  "height": 4
+			  "height": 2
 			},
 			"linkedEntityGuids": null,
 			"visualization": {
 			  "id": "viz.markdown"
 			},
 			"rawConfiguration": {
-			  "text": "![logo](https://www.php.net/images/logos/new-php-logo.svg)\n\nInstrument your application with New Relic - [Add Data](https://one.newrelic.com/launcher/catalog-pack-details.launcher/?pane=eyJuZXJkbGV0SWQiOiJjYXRhbG9nLXBhY2stZGV0YWlscy5jYXRhbG9nLXBhY2stY29udGVudHMiLCJxdWlja3N0YXJ0SWQiOiI0NzVkZWM2OS0xMGM5LTRiYzYtODMxMi0zY2FhMjY2ZmIwMjgifQ==).\n\nDetect common [installation and configuration problems](https://download.newrelic.com/nrdiag/nrdiag_latest.zip) with New Relic agents. Optionally, attach this troubleshooting data to a support case. [Learn more](https://docs.newrelic.com/docs/using-new-relic/cross-product-functions/troubleshooting/new-relic-diagnostics)\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=PHP&entry.358368110=https://onenr.io/0ERzlZYPvQr) here and let us know how we can improve it for you.\n"
+			  "text": "![logo](https://www.php.net/images/logos/new-php-logo.svg)"
```
Comment (pkosanam) [inline dashboards/php/php.json:22]: Request to kindly use the php logo from https://github.com/newrelic/newrelic-quickstarts repository
logo: https://raw.githubusercontent.com/newrelic/newrelic-quickstarts/ea7ba3802665e7c6866dc24609da4fefea0d6d60/quickstarts/php/php/logo.svg

File: dashboards/php/php.json [hunk 2]
```diff
@@ -114,18 +107,34 @@
...
+			  "row": 3,
+			  "width": 2,
+			  "height": 3
+			},
+			"linkedEntityGuids": null,
+			"visualization": {
+			  "id": "viz.markdown"
+			},
+			"rawConfiguration": {
+			  "text": "**About**\n\nInstrument your application with New Relic - [Add Data](https://one.newrelic.com/launcher/catalog-pack-details.launcher/?pane=eyJuZXJkbGV0SWQiOiJjYXRhbG9nLXBhY2stZGV0YWlscy5jYXRhbG9nLXBhY2stY29udGVudHMiLCJxdWlja3N0YXJ0SWQiOiI0NzVkZWM2OS0xMGM5LTRiYzYtODMxMi0zY2FhMjY2ZmIwMjgifQ==).\n\nDetect common [installation and configuration problems](https://download.newrelic.com/nrdiag/nrdiag_latest.zip) with New Relic agents. Optionally, attach this troubleshooting data to a support case. [Learn more](https://docs.newrelic.com/docs/using-new-relic/cross-product-functions/troubleshooting/new-relic-diagnostics)\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=PHP&entry.358368110=https://onenr.io/0ERzlZYPvQr) here and let us know how we can improve it for you.\n"
```
Comment (pkosanam) [inline dashboards/php/php.json:128]: Need modification.
Here, "Detect common installation and configuration problems" is downloading a .zip folder, which is not the ideal scenario

File: dashboards/php/php.json [hunk 3]
```diff
@@ -485,7 +517,7 @@
 			  "nrqlQueries": [
 				{
 				  "accountId": 0,
-				  "query": "SELECT rate(count(apm.service.datastore.operation.duration), 1 minute) FROM Metric WHERE (appName LIKE '%PHP%') FACET `datastoreType`, `operation`, `table` LIMIT 5 TIMESERIES AUTO"
+				  "query": "SELECT average(apm.service.transaction.duration) as responseTime FROM Metric WHERE (appName LIKE '%php%') FACET `appName` LIMIT 20 TIMESERIES AUTO "
```
Comment (pkosanam) [inline dashboards/php/php.json:520]: This query has to be fixed

**Review by pkosanam — CHANGES_REQUESTED** [review]

File: dashboards/php/php.json (hunk 1 above)
Comment (dikshab0202) [reply dashboards/php/php.json]: Done..

**Review by pkosanam — APPROVED** [review]
looks good

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**